erplibre_website_snippets_structures/views/snippets.xml

```diff
@@ -0,0 +1,160 @@
+            <a href="#">
+              <h4 class="mb-0 py-1 text-center position-absolute" style="bottom: 0; left: 0; right: 0; background: rgba(0,0,0,0.5)">Newspaper</h4>
+              <div class="image_overlay"/>
+            </a>
+          </div>
+          <div class="col-lg-4 px-0">
+            <div class="linked_image w-100 px-0 position-relative" style="height: 200px; overflow: hidden; background-image: url('/web/image/website.s_picture_default_image')">
+              <a href="#">
+                <h4 class="mb-0 py-1 text-center position-absolute" style="bottom: 0; left: 0; right: 0; background: rgba(0,0,0,0.5)">Events</h4>
+                <div class="image_overlay"/>
+              </a>
+            </div>
+            <div class="linked_image w-100 px-0 position-relative" style="height: 200px; overflow: hidden; background-image: url('/web/image/website.s_picture_default_image')">
+              <a href="#">
+                <h4 class="mb-0 py-1 text-center position-absolute" style="bottom: 0; left: 0; right: 0; background: rgba(0,0,0,0.5)">Media</h4>
+                <div class="image_overlay"/>
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="row d-flex flex-lg-row flex-column-reverse">
+          <div class="col-lg-4 px-0">
+            <div class="linked_image w-100 px-0 position-relative" style="height: 200px; overflow: hidden; background-image: url('/web/image/website.s_picture_default_image')">
+              <a href="#">
+                <h4 class="mb-0 py-1 text-center position-absolute" style="bottom: 0; left: 0; right: 0; background: rgba(0,0,0,0.5)">Events</h4>
+                <div class="image_overlay"/>
+              </a>
+            </div>
+            <div class="linked_image w-100 px-0 position-relative" style="height: 200px; overflow: hidden; background-image: url('/web/image/website.s_picture_default_image')">
+              <a href="#">
+                <h4 class="mb-0 py-1 text-center position-absolute" style="bottom: 0; left: 0; right: 0; background: rgba(0,0,0,0.5)">Media</h4>
+                <div class="image_overlay"/>
+              </a>
+            </div>
+          </div>
+          <div class="linked_image col-lg-8 col-md-12 px-0 w-100 position-relative" style="height: 400px; overflow: hidden; background-image: url('/web/image/website.s_picture_default_image')">
+            <a href="#">
+              <h4 class="mb-0 py-1 text-center position-absolute" style="bottom: 0; left: 0; right: 0; background: rgba(0,0,0,0.5)">Newspaper</h4>
+              <div class="image_overlay"/>
+            </a>
+          </div>
+        </div>
+      </div>
+    </section>
+  </template>
+
+  <!-- Add snippets to menu -->
+  <template id="add_snippet_structure_to_menu" inherit_id="website.snippets">
+    <xpath expr="//div[@id='snippet_structure']/div[@class='o_panel_body']" position="inside">
```

Comment:
Error-prone use of @Class in view add_snippet_structure_to_menu (): use the hasclass(*classes) function to filter elements by their classes

Reply:
updated to `div[hasclass('o_panel_body')]`